plugins/ui-update/streams.ts

```diff
@@ -0,0 +1,62 @@
+import { BehaviorSubject, Observable, merge, timer, combineLatest } from "rxjs";
+import {
+  filter,
+  map,
+  publishReplay,
+  refCount,
+  switchMap
+} from "rxjs/operators";
+
+import { Package } from "#SRC/js/data/cosmos/Package";
+import { UIMetadata } from "#SRC/js/data/ui-update/UIMetadata";
+
+import { queryCosmosForUIVersions, queryUIServiceForMetadata } from "./queries";
+import { EMPTY_ACTION, UIAction, UIActions } from "./types/UIAction";
+import { versionUpdateAvailable } from "./utils";
+
+const LONG_POLLING_INTERVAL = 300 * 1000;
```

Comment:
Should we pull this from `Config` instead? the current `getRefreshRate` is 2s which is too often but we could create a new value for a longer refresh rate.

Reply:
I am in favour of adding this value in `Config` and taking it from there.

Reply:
+1